.github/workflows/main.yml

```diff
@@ -1,9 +1,14 @@
 name: CI
 on:
   push:
-    branches-ignore: [master]
+    branches:
+      - auto-cargo
+      - try
+      - automation/bors/try
+      - "rust-**"
   pull_request:
-    branches: ['*']
+    branches:
+      - "**"
```

Comment:
Whats the double glob do differently?

I think you can also not have the arry and it works? I think thats what my repos do

Reply:
According to [GitHub Workflow syntax](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet):

- `*`: Matches zero or more characters, but does not match the `/` character. For example, `Octo*` matches `Octocat`.
- `**`: Matches zero or more of any character.

So I think we should use the double glob, unless we have a good reason to exclude branch name with `/` included in them.